examples/image-to-text/README.md

```diff
@@ -35,6 +35,7 @@ Models that have been validated:
   - [meta-llama/Llama-3.2-90B-Vision-Instruct](https://huggingface.co/meta-llama/Llama-3.2-90B-Vision-Instruct)
   - [tiiuae/falcon-11B-vlm](https://huggingface.co/tiiuae/falcon-11B-vlm)
   - [google/paligemma-3b-mix-224](https://huggingface.co/google/paligemma-3b-mix-224)
+  - [THUDM/glm-4v-9b](https://huggingface.co/THUDM/glm-4v-9b)
```

Comment:
we plan to remove this portion also

Reply:
> we plan to remove this portion also

OK, let me remove it.

Reply:
should be added to the tables in the repo's root README.md and and docs/source/index.mdx

Reply:
@mengker33 can you please update asap for next release?

Reply:
Sorry for the late response. I have just updated the doc and fixed the conflicts, please help to review, thank you!